A manpage has a wrong section.
Hyphens were used as minus signs.
Index: ncid-0.86/man/ncid2ncid.1
===================================================================
--- ncid-0.86.orig/man/ncid2ncid.1	2013-02-05 07:39:57.000000000 +0100
+++ ncid-0.86/man/ncid2ncid.1	2013-03-26 19:22:49.897880601 +0100
@@ -1,5 +1,5 @@
 .\" %W% %G%
-.TH NCID2NCID 8
+.TH NCID2NCID 1
 .SH NAME
 .B ncid2ncid
 - Connect multiple NCID servers to a single NCID server
@@ -48,16 +48,16 @@
 The configuration file for \fIncid2ncid\fR is /etc/ncid/ncid2ncid.conf.
 .SH "OPTIONS"
 .TP
-.B -C <filename> | --config <filename>
+.B \-C <filename> | \-\-config <filename>
 Configuration file.
 .br
 Default: /etc/ncid/ncid2ncid.conf
 .TP
-.B -D | --debug
+.B \-D | \-\-debug
 Debug mode, \fIncid2ncid\fR stays attached to the terminal and displays
 all messages that go into the log file.
 .TP
-.B -f <fromhost[:fromport]>
+.B \-f <fromhost[:fromport]>
 Specifies the first sending NCID server.
 It sends the Caller ID and messages to another NCID server.
 Port may be specified by suffixing the hostname with :<port>.
@@ -65,34 +65,34 @@
 .br
 Default: :3333
 .TP
-.B -h | --help
+.B \-h | \-\-help
 Prints this help
 .TP
-.B -L <filename> | --logfile=<filename>
+.B \-L <filename> | \-\-logfile=<filename>
 Specifies the logfile name to use.  The logfile must exist before it is used.
 .br
 Default: /var/log/ncid2ncid.log
 .TP
-.B -t <[host][:port]> | --ncid=<[host][:port]>
+.B \-t <[host][:port]> | \-\-ncid=<[host][:port]>
 Specifies the receiving NCID server.  It receives Caller ID and messages
 from another NCID server.  Port may be specified by suffixing the
 hostname with :<port>.  Input can be <host> or <:port> or <host:port>
 .br
 Default: localhost:3333
 .TP
-.B -P <filename> | --pidfile=<filename>
+.B \-P <filename> | \-\-pidfile=<filename>
 Specifies the pidfile name to write.
 Set to \fI/var/run/ncidd.pid\fR in a rc or init script when used as a service.
 The program will still run if it does not have permission to write a pidfile.
 There is no default.  If pidfile is not set, no pid file will be used.
 Prints this help
 .TP
-.B -V | --version
+.B \-V | \-\-version
 Displays the version
 .TP
-.B -v <1-9> | --verbose <1-9>
+.B \-v <1\-9> | \-\-verbose <1\-9>
 Verbose mode. Send information into the logfile and display information
-for the -D  option.  Set a higher number for more information.
+for the \-D  option.  Set a higher number for more information.
 LEVEL2 will give a warning every time it fails to connect with a NCID server.
 LEVEL8 will print the logfiles from every server, and
 LEVEL9 will add the socket of the received logfile.
@@ -137,15 +137,15 @@
 .SH DIAGNOSTICS
 .nf
     Return Code    Meaning
-    -----------    -------
+    \-\-\-\-\-\-\-\-\-\-\-    \-\-\-\-\-\-\-
          0         Successful
-      -100         Usage
-      -101         Invalid port number
-      -104         Configuration file error
-      -107         Invalid number
-      -108         Missing sending host 1
-      -110         PID file already exists
-        -?         System error
+      \-100         Usage
+      \-101         Invalid port number
+      \-104         Configuration file error
+      \-107         Invalid number
+      \-108         Missing sending host 1
+      \-110         PID file already exists
+        \-?         System error
 .fi
 .SH SEE ALSO
 ncid2ncid.conf.5 ncidd.8, ncidd.conf.5 ncidd.alias.5, ncidd.blacklist
Index: ncid-0.86/man/ncidd.8
===================================================================
--- ncid-0.86.orig/man/ncidd.8	2013-01-27 02:56:43.000000000 +0100
+++ ncid-0.86/man/ncidd.8	2013-03-26 19:22:49.897880601 +0100
@@ -117,48 +117,48 @@
 .SH "OPTIONS"
 .PD 0
 .TP
-.B -A <file> | --alias <file>
+.B \-A <file> | \-\-alias <file>
 Alias file.
 .br
 Default: /etc/ncid/ncidd.alias
 .TP
-.B -B <file> | --blacklist <file>
+.B \-B <file> | \-\-blacklist <file>
 Blacklist file.
 .br
 Default: /etc/ncid/ncidd.blacklist
 .TP
-.B -C <file> | --config <file>
+.B \-C <file> | \-\-config <file>
 Config file.
 .br
 Default: /etc/ncid/ncidd.conf
 .TP
-.B -c <file> | --cidlog <file>
+.B \-c <file> | \-\-cidlog <file>
 Call log data file.
 .br
 Default: /var/log/cidcall.log
 .TP
-.B -D | --debug
+.B \-D | \-\-debug
 Debug mode, \fIncidd\fR stays attached to the terminal and displays
 all messages that go into the log file.
 .TP
-.B -d <file> | --datalog <file>
+.B \-d <file> | \-\-datalog <file>
 Data log data file, usually from a modem.
 .br
 Default: /var/log/datalog.log
 .TP
-.B -e <identifier> | --lineid <identifier>
+.B \-e <identifier> | \-\-lineid <identifier>
 The telephone line identifier is for a modem.  It is normally 1 to 4
 characters.  The default indicator is not displayed, but for one POTS
 line, you could change it to
 .IR POTS .
 For multiple lines you should use a identifier.
 You could change it to the 4 digit extension for an exchange.
-For example, if the telephone number is 321-555-1212 the identifier
+For example, if the telephone number is 321\-555\-1212 the identifier
 would be 1212.
 .br
 Default: -
 .TP
-.B -g <0|1> | --gencid <0|1>
+.B \-g <0|1> | \-\-gencid <0|1>
 Generate a generic Caller ID at ring 2 if one is not received
 from the modem either because the telco is not sending it or
 because the modem does not support it.
@@ -167,71 +167,71 @@
 .br
 Default: gencid = 1
 .TP
-.B -H <0|1> | --hangup <0|1>
+.B \-H <0|1> | \-\-hangup <0|1>
 Automatically hangup up a telephone if the caller name or number is in
 the ncidd.blacklist file.  A hangup text line is sent to the clients.
 .br
 Default: hangup = 0
 .TP
-.B -h | --help
+.B \-h | \-\-help
 Display a help message.
 .TP
-.B -I "string" | --initstr "string"
+.B \-I "string" | \-\-initstr "string"
 Modem initialization string
 .br
 Default: ATE1V1Q0
 .TP
-.B -i "string" | --initcid "string"
+.B \-i "string" | \-\-initcid "string"
 CID initialization string
 .br
 Default: AT+VCID=1
 .br
 if it fails: AT#CID=1
 .TP
-.B -L <file> | --logfile <file>
+.B \-L <file> | \-\-logfile <file>
 Server logfile.
 .br
 Default: /var/log/ncidd.log
 .TP
-.B -l <file> | --lockfile <file>
+.B \-l <file> | \-\-lockfile <file>
 Modem lockfile.
 .br
 Default: /var/lock/LCK..modem
 .TP
-.B -M <MaxBytes> | --cidlogmax <MaxBytes>
+.B \-M <MaxBytes> | \-\-cidlogmax <MaxBytes>
 Set the maximum CID call log file size in bytes.
 .br
 Maximum size is 100000000
 .br
 Default: cidlogmax = 110000
 .TP
-.B -N <0|1> | --noserial <0|1>
+.B \-N <0|1> | \-\-noserial <0|1>
 serial device (0) or no serial device (1)
 .br
 Default: noserial = 0
 .TP
-.B -n <0|1> | --nomodem <0|1>
+.B \-n <0|1> | \-\-nomodem <0|1>
 modem (0) or no modem (1)
 .br
 Default: nomodem = 0
 .TP
-.B -P <pidfile> | --pidfile <pidfile>
+.B \-P <pidfile> | \-\-pidfile <pidfile>
 Server PID file.
 Set to \fI/var/run/ncidd.pid\fR in a rc or init script when used as a service.
 The program will still run if it does not have permission to write a pidfile.
 There is no default.  If pidfile is not set, no pid file will be used.
 .TP
-.B -p <port> | --port <port>
+.B \-p <port> | \-\-port <port>
 Server port.
 .br
 Default: 3333
 .TP
-.B -S <0|1> | --ttyclocal <0|1>
+.B \-S <0|1> | \-\-ttyclocal <0|1>
 Enable (0) or disable (1) modem control signals.
 .br
 Default: modem control signals disabled
 .TP
-.B -s <datatype> | --send <datatype>
+.B \-s <datatype> | \-\-send <datatype>
 Send optional CID data to a client.
 Where \fIdatatype\fR is:
 .br
@@ -245,22 +245,22 @@
 .br
 Default: Optional CID DATA is not sent
 .TP
-.B -T <ttyspeed> | --ttyspeed <ttyspeed>
+.B \-T <ttyspeed> | \-\-ttyspeed <ttyspeed>
 Set the tty port speed to one of: 19200, 9600, 4800
 .br
 Default: ttyspeed = 19200
 .TP
-.B -t <ttyport> | --ttyport <ttyport>
+.B \-t <ttyport> | \-\-ttyport <ttyport>
 Modem device file, or serial port that provides Caller ID information.
 .br
 Default: /dev/modem
 .TP
-.B -V | --version
+.B \-V | \-\-version
 Display the version number.
 .TP
-.B -v <1-9> | --verbose <1-9>
+.B \-v <1\-9> | \-\-verbose <1\-9>
 Verbose mode. Send information into the logfile and display information
-for the -D  option.  Set a higher number for more information.
+for the \-D  option.  Set a higher number for more information.
 Do not use level 9 unless there is a problem in poll().  It grows the
 logfile very fast.
 .br
Index: ncid-0.86/man/sip2ncid.8
===================================================================
--- ncid-0.86.orig/man/sip2ncid.8	2012-09-22 21:21:04.000000000 +0200
+++ ncid-0.86/man/sip2ncid.8	2013-03-26 19:22:49.897880601 +0100
@@ -42,48 +42,48 @@
 The configuration file for \fIsip2ncid\fR is /etc/ncid/sip2ncid.conf.
 .SH "OPTIONS"
 .TP
-.B -C <filename> | --config <filename>
+.B \-C <filename> | \-\-config <filename>
 Configuration file.
 .br
 Default: /etc/ncid/sip2ncid.conf
 .TP
-.B -D | --debug
+.B \-D | \-\-debug
 Debug mode, \fIsip2ncid\fR stays attached to the terminal.
 .TP
-.B -h | --help
+.B \-h | \-\-help
 Prints this help
 .TP
-.B -i <interface> | --interface=<interface>
+.B \-i <interface> | \-\-interface=<interface>
 Specifies the network interface to snoop on.  If this is not specified
 then libpcap will pick a network interface.  This will generally be
 the first Ethernet interface found.
 .TP
-.B -l | --listdevs
+.B \-l | \-\-listdevs
 Returns a list of all network device names that can be used.
 .TP
-.B -L <filename> | --logfile=<filename>
+.B \-L <filename> | \-\-logfile=<filename>
 Specifies the logfile name to use.  The logfile must exist before it is used.
 .br
 Default log filename: /var/log/sip2ncid.log
 .TP
-.B -n <[host][:port]> | --ncid=<[host][:port]>
+.B \-n <[host][:port]> | \-\-ncid=<[host][:port]>
 Specifies the NCID server to connect to.  Port may be specified by
 suffixing the hostname with <:port>, or if you only want to change the
 port, just <:port>.  By default it will connect to port 3333 on
 "127.0.0.1".
 .TP
-.B -P <filename> | --pidfile=<filename>
+.B \-P <filename> | \-\-pidfile=<filename>
 Specifies the pidfile name to write.
 Set to \fI/var/run/ncidd.pid\fR in a rc or init script when used as a service.
 The program will still run if it does not have permission to write a pidfile.
 There is no default.  If pidfile is not set, no pid file will be used.
 .TP
-.B -r <dumpfile> | --readfile <dumpfile>
+.B \-r <dumpfile> | \-\-readfile <dumpfile>
 Read packets from a libpcap capture file instead of the network.
 This also sets the testudp option, verbose to level 3, and no filter
 is applied.  Mostly only useful for development and debug purposes.
 .TP
-.B -s <[host][:port]> | --sip=<[host][:port]>
+.B \-s <[host][:port]> | \-\-sip=<[host][:port]>
 Specifies the hostname of the SIP device to snoop.  You may also specify
 the UDP port by suffixing the hostname with :<port>, or if no hostname
 is wanted, just <:port>.  If you do not specify a host, it defaults to
@@ -91,39 +91,39 @@
 <5061> (Vonage default).  Other Vonage ports are 5060 and 10000.  The
 new Vonage default appears to be <10000>.
 .TP
-.B -T | --testall
+.B \-T | \-\-testall
 Test for all packets.  This option is used to check for IP packets
 without starting the NCID server.  It will display a packet count
 and the packet type.  It will not send any CID data to NCID.
 It also sets the debug option.
 .TP
-.B -t | --testudp
+.B \-t | \-\-testudp
 Test for SIP packets.  This option is used to check if SIP packets
 exist without starting the NCID server.  It will display the Caller
 ID line generated when a call comes in, and a CANCEL line if cancel
 was generated.  It will not send any CID data to NCID.
 It also sets the debug option.
-.B -u | --usage
+.B \-u | \-\-usage
 Prints this help
 .TP
-.B -V | --version
+.B \-V | \-\-version
 Displays the version
 .TP
-.B -v <1-9> | --verbose <1-9>
+.B \-v <1\-9> | \-\-verbose <1\-9>
 Verbose mode. Send information into the logfile and display information
-for the -D  option.  Set a higher number for more information.
+for the \-D  option.  Set a higher number for more information.
 Try LEVEL3 when having a problem.
 .br
 To debug, try: verbose = 3
 .br
 Default: verbose = 1
 .TP
-.B -W <0/1> | --warn <0/1>
+.B \-W <0/1> | \-\-warn <0/1>
 Send 'No SIP packets' or 'SIP packets returned' messages to clients
 .br
 Default: warn = 0
 .TP
-.B -w <dumpfile> | --writefile <dumpfile>
+.B \-w <dumpfile> | \-\-writefile <dumpfile>
 Write packets to a libpcap capture file.
 This also sets the debug option, verbose to level 3, and no filter
 is applied.  Mostly only useful for development and debug purposes.
@@ -151,28 +151,28 @@
 .SH EXAMPLES
 .nf
 # run sip2ncid as root and view status and SIP packets:
-sip2ncid -Dv3
+sip2ncid \-Dv3
 
 # run sip2ncid as root and list all network device names:
-sip2ncid --listdevs
+sip2ncid \-\-listdevs
 
 # run sip2ncid as root in test mode to look for SIP packets.
-sip2ncid --testudp
+sip2ncid \-\-testudp
 
 # run sip2ncid as root in test mode to look for any packets.
-sip2ncid --testall
+sip2ncid \-\-testall
 .fi
 .SH DIAGNOSTICS
 .nf
     Return Code    Meaning
-    -----------    -------
+    \-\-\-\-\-\-\-\-\-\-\-    \-\-\-\-\-\-\-
          0         Successful
-      -100         Usage
-      -101         Invalid port number
-      -104         Configuration file error
-      -107         Invalid number
-      -110         PID file already exists
-        -?         System error
+      \-100         Usage
+      \-101         Invalid port number
+      \-104         Configuration file error
+      \-107         Invalid number
+      \-110         PID file already exists
+        \-?         System error
 .fi
 .SH SEE ALSO
 sip2ncid.conf.5 ncidd.8, ncidd.conf.5 ncidd.alias.5
Index: ncid-0.86/man/yac2ncid.1
===================================================================
--- ncid-0.86.orig/man/yac2ncid.1	2012-10-13 06:49:11.000000000 +0200
+++ ncid-0.86/man/yac2ncid.1	2013-03-26 19:22:49.897880601 +0100
@@ -5,7 +5,7 @@
 - YAC server to NCID server gateway
 .SH SYNOPSIS
 .B yac2ncid
-[-v]
+[\-v]
 .SH DESCRIPTION
 .B YAC2NCID
 acts as a YAC client, listens to the CID information from a YAC server
@@ -35,13 +35,13 @@
 NETCAT_TYPE to CUSTOM in the config file and set custom parameters to
 match your version.
 .PP
-There is also a client output module called \fIncid-yac\fR.  It sends
+There is also a client output module called \fIncid\-yac\fR.  It sends
 the NCID CID information to YAC clients.
 .PD
 .SH "OPTIONS"
 .PD 0
 .TP 10
-.B -v
+.B \-v
 Verbose mode.
 .PD
 .PD
Index: ncid-0.86/man/ncid2ncid.conf.5
===================================================================
--- ncid-0.86.orig/man/ncid2ncid.conf.5	2011-02-28 03:49:28.000000000 +0100
+++ ncid-0.86/man/ncid2ncid.conf.5	2013-03-26 19:22:49.897880601 +0100
@@ -10,7 +10,7 @@
 The \fIncid2ncid.conf\fR file is used to set options.
 Options are set on a command line.
 A line is broken up into words.
-A word is either a string of non-blank characters, everything
+A word is either a string of non\-blank characters, everything
 between double quotes, or an equal sign.
 .PP
 The \fIncid2ncid.conf\fR file understands 3 types of lines:
